File: iniciante/1015/1015.c
#include <stdio.h> // Biblioteca padrão para entrada e saída (scanf, printf)
#include <math.h>  // Biblioteca matemática para funções como sqrt e pow

int main()
{
  float x1, x2, y1, y2, distancia; // Declara as variáveis para coordenadas e distância

  // Lê as coordenadas do primeiro ponto (x1, y1)
  scanf("%f%f", &x1, &y1);

  // Lê as coordenadas do segundo ponto (x2, y2)
  scanf("%f%f", &x2, &y2);

  // Calcula a distância entre os dois pontos usando a fórmula da distância euclidiana
  // sqrt((x2 - x1)^2 + (y2 - y1)^2)
  distancia = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));

  // Imprime a distância com quatro casas decimais
  printf("%.4f\n", distancia);

  return 0; // Finaliza o programa com sucesso
}

File: iniciante/1015/1015.py
import math  # Importa o módulo math, necessário para usar a função sqrt (raiz quadrada)

# Lê dois números reais (float) representando as coordenadas x1 e y1
x1, y1 = map(float, input().split())

# Lê dois números reais (float) representando as coordenadas x2 e y2
x2, y2 = map(float, input().split())

# Calcula a distância entre os dois pontos usando a fórmula da distância Euclidiana:
# distância = √[(x2 - x1)² + (y2 - y1)²]
distancia = math.sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2)

# Imprime a distância com 4 casas decimais
print(f"{distancia:.4f}")
